app/controllers/providers/date_client_told_incidents_controller.rb

```diff
@@ -17,8 +17,9 @@ def incident
 
     def form_params
       merge_with_model(incident) do
+        convert_date_params('incident')
         params.require(:incident).permit(
-          :told_day, :told_month, :told_year, :occurred_day, :occurred_month, :occurred_year
+          :told_on_1i, :told_on_2i, :told_on_3i, :occurred_on_1i, :occurred_on_2i, :occurred_on_3i
```

Comment:
Is this naming format for dates a constraint of the new form builder? `1i`, `2i` and `3i` are just a lot less meaningful than `day`, `month` and `year`.

Reply:
yes unfortunately...I can ask the formbuilder guy about this as it seems like it would be a good thing to change

Reply:
No worries. We can definitely live with it. I was just a bit surprised as it's just not a very Ruby way of naming things

Reply:
Sorry to jump in but I was alerted to this PR and saw the thread.

The reason behind the naming is that [it mimics Rails' existing date functionality](https://guides.rubyonrails.org/v6.0/form_helpers.html#using-date-and-time-form-helpers-model-object-helpers), meaning that there's no need for any extra handling on the app-side. You should be able to just do `params.require(:incident).permit(:told_on, occurred_on)` here and not need to do any jiggery-pokery with the params.